UPGRADE-14.0.md

```diff
@@ -838,6 +838,7 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
         +       ?ArrayObject $context = null,
         +   ): Price
         ```
+    -   FE API: `PaymentStatusDecorator` type has been removed, `Order` type is returned from `UpdatePaymentStatus` instead
```

Comment:
payment status decorator was introduced in this PR, so it's not necessary to mention it in the upgrade notes